puppet/modules/slave/manifests/packaging/debian.pp

```diff
@@ -36,7 +36,8 @@
       arch       => 'amd64',
       release    => 'bookworm',
       apturl     => $debian_mirror,
-      aptcontent => "deb ${debian_mirror} bookworm main non-free contrib\ndeb-src ${debian_mirror} bullseye main non-free contrib\n";
+      aptcontent => "deb ${debian_mirror} bookworm main non-free contrib\ndeb-src ${debian_mirror} bullseye main non-free contrib\n",
+      nodesource => false;
```

Comment:
This is because NodeJS on Debian 12 is new enough so we can use the distro version?

Reply:
commit message says "disable nodesource for bookworm, it ships with NodeJS 18 already", so yes :)